spec/basic/expected_conditions_spec.js

```diff
@@ -0,0 +1,167 @@
+var EC = protractor.ExpectedConditions;
+
+describe('expected conditions', function() {
+  beforeEach(function() {
+    browser.get('index.html#/form');
+  });
+
+  it('should have alertIsPresent', function() {
+    var alertIsPresent = EC.alertIsPresent();
+    expect(alertIsPresent.call()).toBe(false);
```

Comment:
Yea that works too. It's just that in cases where I didn't use an extra variable `expect(EC.alertIsPresent().call())` looked a little more pleasing than `expect(EC.alertIsPresent()())`, but functionally they're the same.